src/web/app/pages-student/student-profile-page/student-profile-page.component.ts

```diff
@@ -110,18 +91,19 @@ export class StudentProfilePageComponent implements OnInit {
         const paramMap: { [key: string]: string } = {
           user: this.user,
           googleid: auth.user.id,
+          entitytype: 'student',
         };
 
         // retrieve profile once we have the student's googleId
-        this.httpRequestService.get('/student/profile', paramMap).subscribe((response: StudentDetails) => {
+        this.httpRequestService.get('/student/profile', paramMap).subscribe((response: StudentProfile) => {
```

Comment:
Under `/services`, we have a student profile service wrapper. Perhaps we could use that?